apis/python/setup.py

```diff
@@ -194,9 +207,10 @@ class BuildExtCmd(build_ext):
     """Builds the Pybind11 extension module."""
 
     def build_extensions(self):
-        opts = ['-std=c++11', '-g']
-        if not TILEDBVCF_DEBUG_BUILD:
-            opts.append('-O2')
+        opts = ['-std=c++11']
```

Comment:
What's the reason for not including debug symbols in the non-debug build? I don't know how much symbols help python extension but for c++ libraries we generally don't strip the symbols out to assist in debugging.

Reply:
Lost in shuffle between main and conda build trees. Restored, along with the `O2`.